### ISSUE TITLE
WebHook incoming object model

### ISSUE THREAD

**Issue by integral-llc** [docusign/recipe-010-webhook-csharp#4]
In your web-hook you are parsing XML on the fly, is there a place where the whole XML is described? I would rather prefer the XML to be de-serialized into a Strongly typed C# POCO object. Do you have such a sample?

**Comment by LarryKlugerDS** [comment]
Hi,
The Connect message XML is documented within the larger [WSDL](https://demo.docusign.net/api/3.0/dsapi.asmx?WSDL).

However, current XML format may optionally include fields that may not be in the WSDL description. (I haven't checked.) In particular, check for the:

* time zone information
* envelope voided reason
* sender account as custom fields
* document fields (tabs)

We are also working on adding adding additional information to either the XML or the planned JSON versions of the notification messages. 

Bottom line: I suggest that you look for the data that you need from the notification message.

Regards,
Larry
